basic_examples/use_cases/basic/after.py

```diff
@@ -0,0 +1,24 @@
+"""Demonstration of calling OpenAI's LLM without Steamship"""
+from steamship import Steamship
+
+from steamship_langchain import OpenAI
+
+client = Steamship()
```

Comment:
wdyt about *always* using the `with Steamship.temporary_workspace() as client:` pattern ?

Reply:
I think that makes a lot of sense. I have been fighting instance caching while writing these, so can imagine others will face the same issues.